src/controller/app_routes.rs

```diff
@@ -202,6 +203,12 @@ impl AppRoutes {
             }
         }
 
+        if let Some(cors) = &ctx.config.server.middlewares.compression {
+            if cors.enable {
```

Comment:
note that this is still `cors`, should be something like `compression`

Reply:
ops copied over :) 
Fixed.